### DISCUSSION TITLE
please help kade engine is crush

### DISCUSSION THREAD

**Discussion by FOXli12** [KadeArchive/Kade-Engine#2365] [Q&A]
I built the game as in the guide, I installed wrzystko but when it is to turn on, in meno 20/21 it crashes as I write lime test windows, it writes in the beginning:
 source / DiffOverview.hx: 65: characters 3-39: Warning: `FlxCamera.defaultCameras` is deprecated, use` FlxG.cameras.setDefaultDrawTarget` instead
source / PlayState.hx: 411: characters 3-39: Warning: `FlxCamera.defaultCameras` is deprecated, use` FlxG.cameras.setDefaultDrawTarget` instead
please, somebody help me. if something is wrong, I apologize because I did google translators

**Comment by isakube** [comment]
kade engine is crush :flushed:

**Reply by FOXli12** [reply]
you know how to fix it?

**Reply by FOXli12** [reply]
please help :(

**Reply by Jotaro-Gaming** [reply]
> you know how to fix it?

i do so basiccly install everything and then download this to download everything https://drive.google.com/file/d/1-10AUdMsXPTU7OSndqnRZeKTdxOatFSO/view?usp=sharing windows only btw

**Reply by FOXli12** [reply]
is not working

**Reply by FOXli12** [reply]
![image](https://user-images.githubusercontent.com/69113665/134782662-cfabb19e-83ad-4f39-947e-da3cffaef072.png)
